samples/web/content/apprtc/js/main.js

```diff
@@ -358,3 +399,142 @@ function toggleFullScreen() {
     trace(event);
   }
 }
+
+roomIdInput.addEventListener('input', function(){
```

Comment:
Absolutely, I was hoping we were planning to do the work you already did. Is that going to be merged soon? I suspect I will have to redo most of what I did to fit in better with the refactored code.